pymeasure/instruments/keithley/keithley6221.py

```diff
@@ -145,6 +155,117 @@ def __init__(self, adapter, name="Keithley 6221 SourceMeter", **kwargs):
         map_values=True,
     )
 
+    ##############
+    # DELTA MODE #
+    ##############
+
+    delta_unit = Instrument.control(
+        ":UNIT:VOLT:DC?", ":UNIT:VOLT:DC %s",
+        """ A string property that controls the reading unit.
+        Valid values are 'V', 'Ohms', 'W' and 'Siemens' """,
+        validator=strict_discrete_set,
+        values={"V": "V", "Ohms": "OHMS", "W": "W", "Siemens": "SIEM"},
+        map_values=True
+    )
+
+    delta_high_source = Instrument.control(
+        ":SOUR:DELT:HIGH?", ":SOUR:DELT:HIGH %g",
+        """ A floating point property that controls the delta high source value in Amps. """,
+        validator=truncated_range,
```

Comment:
Consider using a strict validator, as a truncated validator does not give any feedback that `delta_high_source = -5` does not do, what you expect it to do.